<launch>
	<node pkg="behaviortree_ros" type="nav_through_gate" name="nav_through_gate" output="screen">
        <param name="xml_file" value="$(find behaviortree_ros)/xml/nav_through_gate.xml" />
        <param name="log_file" value="$(find behaviortree_ros)/btlog/nav_through_gate.btlog" />
    </node>
</launch>